### PR TITLE
ADDED MY FORM

### PR DESCRIPTION
**Pull request by El-eric5**
math function added by ERIC

### CODE REVIEW

File: ERIC_FORM.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</head>
+<body>
+  <form>
+    <label>First Name:</label>
+    <input type="text" id="first_name">
+
+    <br>
+
+    <label>Surname:</label>
+  <input type="text" name="" id="Surname">
```
Comment (dellydel) [inline ERIC_FORM.html:18]: No need for the "name" attribute.  You can remove it, especially if it is empty.

File: ERIC_FORM.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+  <form>
+    <label>First Name:</label>
+    <input type="text" id="first_name">
+
+    <br>
```
Comment (dellydel) [inline ERIC_FORM.html:15]: Don't forget the slash here: < br />

**Review by dellydel — CHANGES_REQUESTED** [review]

**Comment by El-eric5** [comment]
eric math function

**Comment by El-eric5** [comment]
i just added math function to the form

File: ERIC_FORM.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   <label for="">Gender:</label><br>
+   <input type="radio" name="gender" id="male">
+   <label for="male">Male:</label>
+    <br>
+    <input type="radio" name="gender" id="female">
+    <label for="female">Female:</label><br>
+    <br>
+    <input type="button" value="Click Me!"onclick="MathFunction(20,2)" >
+
```
Comment (dellydel) [inline ERIC_FORM.html:41]: Lots of extra space here, clean up the extra lines a bit here

File: Eric_maths.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+ function MathFunction(eric, agu){
+    // set 2 variables
+    //let eric =20;
+    //let agu =2;
+
+    console.log(eric/agu);
+
```
Comment (dellydel) [inline Eric_maths.js:7]: Remove extra spaces here

**Review by dellydel — CHANGES_REQUESTED** [review]
Just some small changes and you are good to go... Don't forget to add the slash to your line break elements: <br / >

**Review by dellydel — APPROVED** [review]

**Pull request merged**